Obsidian.API/_Interfaces/IBanList.cs

```diff
@@ -0,0 +1,13 @@
+namespace Obsidian.API;
+
+public interface IBanList
+{
+    public void AddBan(IPlayer player, int duration);
+    public void AddBan(string username, int duration);
+    public void RemoveBan(IPlayer player);
+    public void RemoveBan(string username);
+    public IBan GetBan(IPlayer player);
+    public IBan GetBan(string username);
+    public bool IsBanned(IPlayer player);
+    public bool IsBanned(string username);
+}
```

Comment:
Would like if you could iterate over a list, like usual.

Reply:
would List<IBan> work?

Reply:
`IList<IBan>` would be better, since this is already an interface.